android/src/main/java/com/instabug/instabugflutter/OnMethodCallTests.java

```diff
@@ -0,0 +1,148 @@
+package com.instabug.instabugflutter;
+
+import android.support.annotation.Nullable;
+
+import org.mockito.Mockito;
+import org.mockito.Spy;
+
+import java.util.ArrayList;
+
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel.Result;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+ class OnMethodCallTests {
```

Comment:
Is there a way to exclude this class from being included when the plugin is installed?